Tools/TestRunnerShared/Bindings/JSBasics.h

```diff
@@ -35,6 +35,8 @@ namespace WTR {
 std::optional<bool> toOptionalBool(JSContextRef, JSValueRef);
 std::optional<double> toOptionalDouble(JSContextRef, JSValueRef);
 
+bool valueIsUndefinedOrNull(JSContextRef, JSValueRef);
```

Comment:
This doesn't seem to be implemented anywhere?

Reply:
(I do have a slight preference for `!valueIsUndefinedOrNull()` over `isValidValue`, since the former is more explicit)

Reply:
Yes, that seems like a good change!

Reply:
I moved away from `valueIsUndefinedOrNull()` because the JSValue can be null (which is different from `JSValueIsNull()`).